src/commands/synthetics/reporters/default.ts

```diff
@@ -323,9 +327,38 @@ export class DefaultReporter implements MainReporter {
 
     if (summary.batchId) {
       const batchUrl = getBatchUrl(baseUrl, summary.batchId)
-      lines.push('View test batch in Continuous Testing Explorer: ' + chalk.dim.cyan(batchUrl))
+      lines.push('View full summary in Datadog: ' + chalk.dim.cyan(batchUrl))
+    }
+    lines.push(`\n${b('Continuous Testing Summary:')}`)
+    lines.push(`Test Results: ${runSummary.join(', ')}${extraInfoStr}`)
+
+    if (orgSettings && orgSettings.orgMaxConcurrencyCap > 0) {
+      lines.push(
+        `Max parallelization configured: ${orgSettings.orgMaxConcurrencyCap} test${
+          orgSettings.orgMaxConcurrencyCap > 1 ? 's' : ''
+        } running at the same time`
+      )
+    }
+
+    if (this.totalDuration) {
+      const min = Math.floor(this.totalDuration / (60 * 1000))
+      const sec = Math.round((this.totalDuration % (60 * 1000)) / 1000)
+      lines.push(
+        `Total Duration:${min > 0 ? ' ' + min.toString() + 'm' : ''}${sec > 0 ? ' ' + sec.toString() + 's' : ''}`
+      )
+    }
+
+    if (
+      orgSettings &&
+      typeof orgSettings.orgMaxConcurrencyCap !== 'undefined' &&
+      orgSettings.orgMaxConcurrencyCap !== 1
```

Comment:
To confirm with @peculiarism when we want to show this message but I think everytime `orgMaxConcurrencyCap > 0` ?

Reply:
@peculiarism confirmed the message should also take place for the case with a cap of 1